### DISCUSSION TITLE
Theme support in the dashboard (alpha.nvim) ?

### DISCUSSION THREAD

**Discussion by saurabhya** [NvChad/NvChad#874] [Q&A]
Despite the availability of a wide range of themes, why the startup page (alpha.nvim) doesn't support theme colors.
It would be really nice to have that since the startup page seems a little bland. Is there a particular reason for this?

**Comment by siduck** [comment]
you can override the highlights and add your colors. Yes I wanted to keep it basic so i just used grayish colors

**Reply by saurabhya** [reply]
I wanted to add theme colors to the start page which changes according to the theme. Is there a way to do that?
I do not know Lua so facing a hard time doing that.

**Reply by siduck** [reply]
override default highlights first ( check our docs , config section in sidebar)
